modules/primer-marketing-utilities/lib/margin.scss

```diff
@@ -0,0 +1,80 @@
+// Margin spacer utilities
+// stylelint-disable block-opening-brace-space-before, declaration-colon-space-before, primer/selector-no-utility, comment-empty-line-before
+@for $i from 1 through length($spacers) {
+  $size: #{nth($spacers, $i)};
+  $scale: #{$i - 1};
+
+  /* Set a #{$size} margin to all sides */
+  .m-#{$scale}  { margin       : #{$size} !important; }
+  /* Set a #{$size} margin on the top */
+  .mt-#{$scale} { margin-top   : #{$size} !important; }
+  /* Set a #{$size} margin on the right */
+  .mr-#{$scale} { margin-right : #{$size} !important; }
```

Comment:
Do we need margin right and left for these? I can't think of any situations where I've wanted bigger values for these than what we've got. Currently, I feel like we just need the vertical values (to allow us to refactor jumbotron and featurette custom css). Taking these out will reduce a lot of final CSS.

(I'd be curious to hear from others on @primer/design-systems on if they agree)

Reply:
im gonna remove them unless others think otherwise! @primer/design-systems